src/helpers/user/user-helper.js

```diff
@@ -8,6 +9,145 @@ const principalStatusApiMap = {
   Inactive: 'disabled',
   All: 'all',
 };
+
+const getBaseUrl = (url) => {
```

Comment:
Note. We should be feeding it from chrome service and configure a env variable in the OC namespace. @florkbr can you create a ticket for this?

Reply:
@Hyperkid123 done! https://issues.redhat.com/browse/RHCLOUD-34107